MemoryLeak.Dockerfile

```
@@ -1,23 +1,27 @@
 FROM node:23.11.1-alpine3.21
 
 RUN apk add --no-cache \
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)